static/js/src/certified-search-results.js

```diff
@@ -2,8 +2,17 @@
 
 function clearFilters() {
   hideDrawerPageReload();
+
   let objUrl = new URL(window.location);
-  objUrl.search = "";
+  const href = window.location.href;
```

Comment:
you can do `const { href } = window.location;` if you want to be cool, but this is fine.

Reply:
I am not sure if we are compiling this "new JS" to "old JS", but appartenly this doesn't work for IE 11. So this comment is optional, just FYI.